packages/editor/src/MessageInputEditor.tsx

```diff
@@ -75,8 +75,9 @@ export const MessageInputEditor = () => {
       style={{
         overflow: 'auto',
         height: 'auto',
-        // making this explicit
+        minHeight: 32,
```

Comment:
Is this value used anywhere else / referred to anywhere else / anything we can comment up?